### PR TITLE
add snipe & editsnipe commands + categorize cache + other stuff lol

### PR DESCRIPTION
**Pull request by caneleex**

### CODE REVIEW

File: src/main/java/de/anteiku/kittybot/events/OnGuildMessageEvent.java [hunk 1]
```diff
@@ -18,6 +23,9 @@
 
 	@Override
 	public void onGuildMessageReceived(GuildMessageReceivedEvent event){
+		var content = event.getMessage().getContentRaw();
+		if (!content.isEmpty())
```
Comment (TopiSenpai) [inline src/main/java/de/anteiku/kittybot/events/OnGuildMessageEvent.java:27]: {} :)

Comment (caneleex) [reply src/main/java/de/anteiku/kittybot/events/OnGuildMessageEvent.java:27]: you can just format the code with your code style then

**Review by TopiSenpai — APPROVED** [review]
lgtm I guess formatting then later

File: src/main/java/de/anteiku/kittybot/events/OnGuildMessageEvent.java (hunk 1 above)
Comment (TopiSenpai) [reply src/main/java/de/anteiku/kittybot/events/OnGuildMessageEvent.java:27]: yus

**Review by TopiSenpai — APPROVED** [review]
lgtm

**Pull request merged**